index.js

```diff
@@ -153,6 +157,23 @@ class FastifyOtelInstrumentation extends InstrumentationBase {
           return
         }
 
+        if (routeOptions.otel === false) {
+          instrumentation.logger.debug(
+            `Ignoring route instrumentation ${routeOptions.method} ${routeOptions.url} because it is disabled`
+          )
```

Comment:
```suggestion
          instrumentation.logger.debug(
            'Ignoring route instrumentation %s %s because it is disabled', routeOptions.method, routeOptions.url
          )
```

Reply:
I tried to be consistent with 

https://github.com/fastify/otel/blob/cc5fdf9402849b659952d1675d053dc856b933b6/index.js#L155

Should I still change it?

Reply:
No, not needed. It is not the fastify logger 

What is this `logger` @metcoder95 ?